tests/test_eigen_tensor.py

```diff
@@ -0,0 +1,243 @@
+import sys
+
+import pytest
+
+np = pytest.importorskip("numpy")
+eigen_tensor = pytest.importorskip("pybind11_tests.eigen_tensor")
+submodules = [eigen_tensor.c_style, eigen_tensor.f_style]
+try:
+    from pybind11_tests import eigen_tensor_avoid_stl_array as avoid
+
+    submodules += [avoid.c_style, avoid.f_style]
+except ImportError:
+    print("Could not load avoid stl array tests", sys.stderr)
```

Comment:
Hmm, having a print statement here is not ideal. Normally I would look into pytest.importorskip. Use the pytest logger or whatever proper way exists to print out the info.